README.md

```diff
@@ -129,6 +132,15 @@ optionally sets the hostname/address for *revgeoc*; defaults to `127.0.0.1` and
 REQUIREMENTS
 ============
 
+docker
+-----
+Create a .env file with an OPENCAGE_APIKEY or LOCATIONIQ_APIKEY variable defined.
```

Comment:
where (in which path) must this .env be created? alongside the docker-compose file? Please say so.